src/lamplib/src/genny/tasks/mothra_service.py

```diff
@@ -0,0 +1,58 @@
+import os
+from typing import NamedTuple, Optional
+from unicodedata import name
+from dotenv import load_dotenv
+from github import Github
+from github import Auth
+from github.ContentFile import ContentFile
+import yaml
+
+MOTHRA_REPO = "10gen/mothra"
+TEAMS_DIR = "mothra/teams"
+TEAMS_FILES = ["cloud.yaml", "database.yaml", "rnd_dev_prod.yaml", "star.yaml"]
+TEAMS_FILES_PATHS = [f"{TEAMS_DIR}/{file}" for file in TEAMS_FILES]
+
+
+class Team(NamedTuple):
+    name: str
+    support_slack_channel_name: Optional[str] = None
+    support_slack_channel_id: Optional[str] = None
+
+
+class MothraService:
+
+    def __init__(self):
+        self.team_map = self._load_team_map()
+
+    def get_team(self, team_name: str) -> Optional[Team]:
+        return self.team_map.get(team_name, None)
+
+    def _load_team_map(self) -> dict[str, Team]:
+        github = Github(auth=Auth.Token(self._get_github_token()))
+        repo = github.get_repo(MOTHRA_REPO)
```

Comment:
Question: I'm ignorant in this but isn't this just a fancy git clone ? If so then could we not just clone the repo and avoid the github API?

Reply:
Personally I think simpler is better ... but you have documented how to get a token and shown that it's unlikely that we'll ever hit the API limit ... so I'm happy to go with either.